lib/emque/producing/message/message_with_changeset.rb

```diff
@@ -30,15 +46,31 @@ class ChangesPayloadGenerator
       def initialize(opts = {})
         @original = opts.fetch(:original, {}) || {}
         @updated = opts.fetch(:updated, {}) || {}
+        @translated_attrs = opts.fetch(:translated_attrs, {}) || {}
```

Comment:
we've standardized fetch to the following syntax `opts.fetch(:translated_attrs) { {} }` because the block is evaluated each time vs once. Not sure you need the `|| {}` here either.